### PR TITLE
add sql.js@0.3.2 w/ npm autoupdate

### LINKED ISSUE #8449: [Request] Add SQL.js
**Library name:** SQL.js
**Git repository url:** https://github.com/kripken/sql.js/
**npm package url(optional):** 
**License(s):** MIT licensed
**Official homepage:** http://kripken.github.io/sql.js/

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #8449 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kripken/sql.js/
 * Official website (optional, not the repository): 
 * NPM package url (optional): https://www.npmjs.com/package/sql.js
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 2996
   - Count of watchers: 164
   - Count of forks: 328
 * NPM download stats (optional):
   - Downloads in the last day: 49
   - Downloads in the last week: 201
   - Downloads in the last month: 1044

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8449, cc @kripken

### CODE REVIEW

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

File: ajax/libs/sql.js/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "type": "git",
+    "url": "http://github.com/kripken/sql.js.git"
+  },
+  "author": "Alon Zakai",
+  "npmName": "sql.js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "js/*sql!(*raw)*"
```
Comment (PeterDaveHello) [inline ajax/libs/sql.js/package.json]: @pvnr0082t @x09326 what's the reason why to drop `*raw` here?

Comment (PeterDaveHello) [inline ajax/libs/sql.js/package.json]: The raw files looks not useful here, @pvnr0082t could you please confirm with the author about the files should be hosted here?

**Comment by pvnr0082t** [comment]
@x09326 @PeterDaveHello I confirmed that the raw files don't need to be added. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t would you please rebase your branch with the latest master branch? Thanks.

**Comment by pvnr0082t** [comment]
@x09326 I have rebase the branch with the latest master branch. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@lovasoa looks to be the most active author, @pvnr0082t you could also tag multi authors next time.

**Pull request merged**